### PR TITLE
contrib: minor spelling tweaks

### PR DESCRIPTION
**Pull request by brettkoonce**

### CODE REVIEW

File: tensorflow/contrib/tensor_forest/kernels/v4/decision_node_evaluator.h [hunk 1]
```diff
@@ -96,7 +96,7 @@ std::unique_ptr<DecisionNodeEvaluator> CreateDecisionNodeEvaluator(
 std::unique_ptr<DecisionNodeEvaluator> CreateBinaryDecisionNodeEvaluator(
     const decision_trees::BinaryNode& node, int32 left, int32 right);
 
-struct CandidateEvalatorCollection {
+struct CandidateEvaluatorCollection {
```
Comment (brettkoonce) [inline tensorflow/contrib/tensor_forest/kernels/v4/decision_node_evaluator.h]: is there a good way to check this?  it does not appear anywhere in the primary codebase.

**Review by sb2nov — APPROVED** [review]

**Comment by sb2nov** [comment]
@brettkoonce can you resolve the merge conflicts.

**Comment by brettkoonce** [comment]
@sb2nov rebased, thanks!

File: tensorflow/contrib/tensor_forest/kernels/v4/decision_node_evaluator.h (hunk 1 above)
Comment (sb2nov) [reply tensorflow/contrib/tensor_forest/kernels/v4/decision_node_evaluator.h]: Can we rollback this change from this PR as this is a breaking change.

We can push the other documentation changes for now and deal with this independently.

**Review by sb2nov — CHANGES_REQUESTED** [review]
Rollback the class name change

**Comment by brettkoonce** [comment]
@sb2nov rolled back, let me know if you need anything else!

**Review by sb2nov — APPROVED** [review]

**Comment by brettkoonce** [comment]
@sb2nov rebased, please try the tests again!

**Comment by brettkoonce** [comment]
@benoitsteiner just rebased to master/HEAD again, for what it's worth!

**Review by sb2nov — APPROVED** [review]

**Comment by brettkoonce** [comment]
@sb2nov one more try!

**Review by sb2nov — APPROVED** [review]

**Comment by brettkoonce** [comment]
@sb2nov is there anything i can do to help fix things?

**Pull request merged**